org-fc-review.el

```diff
@@ -66,6 +66,13 @@
   :type 'hook
   :group 'org-fc)
 
+(defcustom org-fc-review-hide-title-in-header-line nil
+  "Whether or not to hide the file title in review header line.
+
+Hide title for individual cards by adding the :nottile: tag."
```

Comment:
There's a small type here, `:nottile:` should be `:notitle:`

Reply:
Thank you for catching this!